<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgBulkCharge</class>
 <widget class="QDialog" name="CDlgBulkCharge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>211</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>314</width>
    <height>211</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>314</width>
    <height>211</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>月租卡批量续期</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>291</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>批量处理条件</string>
   </property>
   <widget class="QLineEdit" name="edtSelfStart">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>21</y>
      <width>84</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbSelfNo">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>23</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>自编号从</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtSelfEnd">
    <property name="geometry">
     <rect>
      <x>196</x>
      <y>21</y>
      <width>84</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>21</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>到</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbEndDate">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>53</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>截至日期从</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtEndStart">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>51</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtEndEnd">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>51</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>51</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>到</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxUnit">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>81</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbUnit">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>83</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>单位</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>180</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="btnOk">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widgetDate" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>122</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>176</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>新的截至日期</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dtNewEnd"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetState" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>153</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>299</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="chkEnterMustCard">
       <property name="text">
        <string>入场必须刷卡</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkLeaveMustCard">
       <property name="text">
        <string>出场必须刷卡</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkMIMO">
       <property name="text">
        <string>允许多进多出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
